homeassistant/components/media_player/__init__.py

```diff
@@ -214,6 +213,25 @@ class MediaPlayerDeviceClass(StrEnum):
     ATTR_MEDIA_REPEAT,
 ]
 
+# mypy: disallow-any-generics
+
+
+class CacheImage(TypedDict, total=False):
+    """Class to hold a cached image."""
+
+    lock: asyncio.Lock
+    content: tuple[bytes | None, str | None]
+
+
+class ImageCache(TypedDict):
+    """Class to hold a cached image."""
+
+    images: collections.OrderedDict[str, CacheImage]
+    maxsize: int
+
+
+ENTITY_IMAGE_CACHE = ImageCache(images=collections.OrderedDict(), maxsize=16)
```

Comment:
Just wondering should all of these get a `_` prefix? They are only used within this file after all.